src/data/unitOptions.json

```diff
@@ -23,6 +23,18 @@
           "restricted_upgrades": []
         }
       ],
+      "Operative": [
+        {
+          "name": "Chewbacca",
+          "cost": 165,
+          "rank": "commander",
+          "faction": "rebel",
+          "type": "trooper",
+          "keywords": "",
+          "upgrades_slots": ["force", "force", "leadership"],
```

Comment:
Chewy can use the force????

Reply:
HAHA data TBD

Reply:
Instant googled "chewbacca force-sensitive" in case somehow relevant in Rise Of Skywalker